cpp/tests/copying/get_value_tests.cpp

```diff
@@ -301,7 +306,9 @@ TYPED_TEST(ListGetFixedWidthValueTest, NestedGetNonNullEmpty)
 
 TYPED_TEST(ListGetFixedWidthValueTest, NestedGetNull)
 {
-  using LCW = cudf::test::lists_column_wrapper<TypeParam, int32_t>;
+  using LCW      = cudf::test::lists_column_wrapper<TypeParam, int32_t>;
+  using FCW      = cudf::test::fixed_width_column_wrapper<TypeParam>;
+  using offset_t = cudf::test::fixed_width_column_wrapper<size_type>;
```

Comment:
Minor nitpick: I was informed that we are to use `offset_type` instead of `size_type` for offset columns. (This might well be a distinction without a difference at the moment.) 

I will need to change my code to incorporate this as well.

Reply:
Thanks! I definitely was not aware.